tf_dataset/pair_generator.py

```diff
@@ -8,7 +8,7 @@ class PairGenerator(object):
     person2 = 'person2'
     label = 'same_person'
 
-    def __init__(self, lfw_path='resources' + os.path.sep + 'lfw'):
+    def __init__(self, lfw_path='./tf_dataset/resources' + os.path.sep + 'lfw'):
```

Comment:
That's on OK change but doesn't really solve the path issue - you just made an assumption of the present working directory from which this line is going to get invoked. 

If you want to really make it robust, we can use relative path like e.g. `os.path.abspath(__file__)` and `us.path.join` to get point to resources relative to `pair_generator.py`